src/domain/cart/CartPage.tsx

```diff
@@ -1,5 +1,39 @@
+import classes from "./../../styles/cartPage.module.css";
+import { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+import { CartCard } from "./CartCard";
+import { cartSelector } from "./cartSlice";
+
 export const CartPage = (): JSX.Element => {
+  const [total, setTotal] = useState<number>(0);
+  const cart = useSelector(cartSelector);
+
+  useEffect(() => {
+    setTotal(cart.reduce((acc, curr) => (acc += curr.price), 0));
+  }, [total,cart]);
```

Comment:
No need for the state and useEffect. Extract the total fn in an another file and call it here and cache its value.
`const total = getTotalCartPrice(cartItems)`